src/styles/blocks/navigation.scss

```diff
@@ -0,0 +1,45 @@
+.navigation {
+  &__list {
+    margin: 0;
+    padding: 0;
+    display: flex;
+    list-style-type: none;
+    align-items: end;
```

Comment:
The value 'end' for 'align-items' is not valid. You might want to use 'flex-end' instead, as 'align-items' accepts values like 'flex-start', 'center', 'flex-end', etc.